content/page/get-involved.md

```diff
@@ -0,0 +1,63 @@
+
+* [Subscribe to our regular OBF newsletter](https://mailman.open-bio.org/mailman/listinfo/open-bio-l/) (and if you have news to share, you can [also suggest topics](https://github.com/OBF/newsletter/issues).
+* You can [join our community _Slack_](https://join.slack.com/t/open-bio/shared_invite/zt-1pnswao9y-8igcckVxBXhQHCMweHt_NA), where you will find fellow open bioinformatics folks to chat with.
+* We also [have an open _LinkedIn_ group](https://www.linkedin.com/groups/9539620/) to get in touch with folks, find interesting job opportunities and more.
+* Last but not least, we also have active presences on social media, on both [BlueSky](https://bsky.app/profile/openbio.bsky.social) and [Mastodon](https://genomic.social/@openbio).
+
+## Come to our in-person and virtual events
+
+We organize an annual conference, the [_Bioinformatics Open Source Conference_](https://www.open-bio.org/events/bosc) (BOSC) and the associated [_Collaboration Fest_](https://www.open-bio.org/events/bosc-2025/ismb-collaborationfest-2025/).
+These events are a great way to meet others engaged in open source/science around biology.
+BOSC features talks, posters and panel discussions on the latest research in open bioinformatics, while 
+Collaboration Fest offers the chance to work together with others on improving open bioinformatics tools, by coding but also writing documentation and other non-coding tasks.
+
+
+# Applying to become a voting member of OBF
+
+You can also apply to become a voting member of the Open Bioinformatics Foundation, which allows voting on larger OBF issues. 
+To be considered as a voting member, you must already be actively and demonstrably engaged in open bioinformatics work in some form.
+Examples of such engagement could include contributing to existing open bioinformatics projects, publishing in the field, or attending open source bioinformatics conferences such as BOSC.
+The reason for this requirement is that we expect members to have a long-term engagement with both the field and the OBF itself.
+The OBF requires a quorum for voting, so we want OBF members to remain actively involved.
+
+The best time and place to join is at the annual [BOSC](/events/bosc/) conference, because attendees are automatically deemed eligible based on the fact that they attended BOSC, but you can apply for membership at any time. 
+
+![](/wp-content/uploads/2019/02/obf_membership.jpg)
+
+## History and Bylaws
+
+The OBF membership body was formally established at the 2005 Board of Directors meeting. As laid out in the [OBF Bylaws](https://github.com/OBF/obf-docs/blob/master/OBF%20Bylaws.md), officers in the Board of Directors will be elected from the nominees by the membership, and candidates for future Directors will be nominated from the membership when seats are added or a term expires.
```

Comment:
I'm not sure that gives quite the right impression for Board Officer elections, but can't immediately think of how to clarify it.

Reply:
Looking at the bylaws this seems like it's in theory how the process works, in practice I'm not sure if we ever had the membership at large suggest candidates? Tried to suggest a slight rewording to make it clearer:

```suggestion
The OBF membership body was formally established at the 2005 Board of Directors meeting. As laid out in the [OBF Bylaws](https://github.com/OBF/obf-docs/blob/master/OBF%20Bylaws.md), the Board of Directors will elects Directors when terms expire or new seats are added from the pool of candidates that the membership nominated.
```

Reply:
That's what I tried to convey in my suggested change above, do you think there's a better/clearer way to put that?

Reply:
Gotcha, edited it to be that!